src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1659,7 +1659,7 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
                 }
             });
         }
-        
+
```

Comment:
i don't know why...in my vs diff tool this changes weren't visible.

Reply:
Yes, please remove any changes that aren't part of the PR.